Includes/menu.hpp

```diff
@@ -7,6 +7,8 @@
 #include <vector>
 #include <string>
 
+// TODO: Calculate this based off screen size?
+#define MENU_PAGE_SIZE 10
```

Comment:
Value is available as  `size_t Menu::itemsToShow` - maybe make it an argument for the `pageup()`/`pagedown()` functions?

Reply:
Awesome, changed!